pylops/LinearOperator.py

```diff
@@ -82,6 +82,7 @@ def __init__(
         dimsd=None,
         clinear=None,
         explicit=None,
+        flattened=None,
```

Comment:
Seems like you have not added flattened to the doctoring?

Reply:
Done